src/python.rs

```diff
@@ -41,6 +42,96 @@ enum ResamplingFunctionF32 {
     Count,
 }
 
+#[pymethods]
+impl ResamplingFunctionF32 {
+    #[new]
+    fn new(value: i32) -> PyResult<Self> {
+        value.try_into()
+    }
+
+    #[classmethod]
+    fn values(_cls: &Bound<'_, PyType>) -> Vec<i32> {
+        vec![
+            Self::Average.value(),
+            Self::Sum.value(),
+            Self::Max.value(),
+            Self::Min.value(),
+            Self::Last.value(),
+            Self::Count.value(),
+        ]
+    }
+
+    #[classmethod]
+    fn members(_cls: &Bound<'_, PyType>) -> Vec<(String, i32)> {
+        vec![
+            (Self::Average.to_string(), Self::Average.value()),
+            (Self::Sum.to_string(), Self::Sum.value()),
+            (Self::Max.to_string(), Self::Max.value()),
+            (Self::Min.to_string(), Self::Min.value()),
+            (Self::Last.to_string(), Self::Last.value()),
+            (Self::Count.to_string(), Self::Count.value()),
+        ]
+    }
+
+    fn __str__(&self) -> String {
+        format!("{}.{}", "ResamplingFunction", self)
+    }
+
+    fn __repr__(&self) -> String {
+        self.__str__()
```

Comment:
The canonical `repr()` for `Enum` is:

```suggestion
        format!("<{}.{}: {}>", "ResamplingFunction", self.to_string(), self)
```

Maybe we can use that instead?

Reply:
I would change it to:
```rust
fn __repr__(&self) -> String {
    format!("<{}: {}>", self.__str__(), self.value())
}
```

which would do this: 
```python
repr(ResamplingFunction.Average) == "<ResamplingFunction.Average: 0>"
```

Reply:
Yeah, I was about to suggest that but didn't know what's the most common approach in rust :)